Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -17,7 +17,10 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.datastore.Key;
```

Comment:
maybe put this file and related files under a separate folder DB along with the test file

Reply:
ok